scripts/soltest.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPO_ROOT="$(dirname "$0")"/..
+USE_DEBUGGER=0
+DEBUGGER="gdb --args"
+BOOST_OPTIONS=
+SOLTEST_OPTIONS=
+
+while [ $# -gt 0 ]
+do
+	case "$1" in
+		--debugger)
+			shift
+			DEBUGGER="$1"
+			USE_DEBUGGER=1
+			;;
+		--debug)
+			USE_DEBUGGER=1
+			;;
+		--boost-options)
+			shift
+			BOOST_OPTIONS="${BOOST_OPTIONS} $1"
+			;;
+		-t)
+			shift
+			BOOST_OPTIONS="${BOOST_OPTIONS} -t $1"
+			;;
+		--show-progress)
+			BOOST_OPTIONS="${BOOST_OPTIONS} $1"
+			;;
+		-p)
```

Comment:
This and the previous case can be combined.

Reply:
Didn't know that was possible via ``;&`` in bash scripts - done.

Reply:
Nevermind an or-case is much better anyways...